core/src/Streamly/Internal/Data/Unboxed.hs

```diff
@@ -297,8 +261,12 @@ class Storable a => Unbox a where
     -- IMPORTANT: The implementation of this interface may not check the bounds
     -- of the array, the caller must not assume that.
     pokeByteIndex :: MutableByteArray a -> Int -> a -> IO ()
+    -- | Get the size.
+    sizeOf :: a -> Int
+    -- | Get the alignment.
+    alignment :: a -> Int
```

Comment:
we can remove alignment for now, if required we can add it later.

Reply:
Removing this would make the code rather unclean. We use alignment in unboxed pinned arrays.
We can probably ignore alignment as a whole I guess later if it does not serve any benefit.